exercises/practice/two-bucket/.meta/example.php

```diff
@@ -0,0 +1,157 @@
+<?php
+
+declare(strict_types=1);
+
+class TwoBucket
+{
+    private int $goal;
+    private array $buckets;
+    public int $moves;
+    public string $goalBucket;
+    public int $otherBucket;
```

Comment:
As we have only one shot for the interface, let's make the property names very clear:

```suggestion
    public int $numberOfActions;
    public string $nameOfBucketWithDesiredLiters;
    public int $litersLeftInOtherBucket;
```

About the first value: the instructions talk about actions, not moves. On the second value: the word goal doesn't appear in the instructions, they are always about "desired liters" or "exact liters". And the last values is the liters, not the number of the bucket. I hope that's clear enough for any student.